From d8a89f98d9337f2a930790ee19868b2d03c449a7 Mon Sep 17 00:00:00 2001
From: Hariprasad Kelam <hkelam@marvell.com>
Date: Thu, 26 Nov 2020 12:47:19 +0530
Subject: [PATCH 0944/1921] octeontx2-pf: cn10k: export APIs required by VF
 driver

export APIS required by virtual function driver.

Change-Id: I5921eda0db9c8e3e9c1272121cd78f701004a48d
Signed-off-by: Hariprasad Kelam <hkelam@marvell.com>
Reviewed-on: https://sj1git1.cavium.com/c/IP/SW/kernel/linux/+/41076
Tested-by: sa_ip-sw-jenkins <sa_ip-sw-jenkins@marvell.com>
Reviewed-by: Sunil Kovvuri Goutham <Sunil.Goutham@cavium.com>
[WK: The original patch got from Marvell sdk11.21.09]
Signed-off-by: Wenlin Kang <wenlin.kang@windriver.com>
---
 drivers/net/ethernet/marvell/octeontx2/nic/cn10k.c       | 1 +
 drivers/net/ethernet/marvell/octeontx2/nic/otx2_common.c | 1 +
 2 files changed, 2 insertions(+)

diff --git a/drivers/net/ethernet/marvell/octeontx2/nic/cn10k.c b/drivers/net/ethernet/marvell/octeontx2/nic/cn10k.c
index 1b5a292a59b0..0bc718b774c8 100644
--- a/drivers/net/ethernet/marvell/octeontx2/nic/cn10k.c
+++ b/drivers/net/ethernet/marvell/octeontx2/nic/cn10k.c
@@ -84,6 +84,7 @@ int cn10k_vf_lmtst_init(struct otx2_nic *vf)
 	vf->nix_lmt_size = vf->nix_lmt_lines * LMT_LINE_SIZE;
 	return 0;
 }
+EXPORT_SYMBOL(cn10k_vf_lmtst_init);
 
 int cn10k_sq_aq_init(void *dev, u16 qidx, u16 sqb_aura)
 {
diff --git a/drivers/net/ethernet/marvell/octeontx2/nic/otx2_common.c b/drivers/net/ethernet/marvell/octeontx2/nic/otx2_common.c
index 717f19060444..ac543b93c74d 100644
--- a/drivers/net/ethernet/marvell/octeontx2/nic/otx2_common.c
+++ b/drivers/net/ethernet/marvell/octeontx2/nic/otx2_common.c
@@ -1677,6 +1677,7 @@ u16 otx2_get_max_mtu(struct otx2_nic *pfvf)
 	}
 	return max_mtu;
 }
+EXPORT_SYMBOL(otx2_get_max_mtu);
 
 #define M(_name, _id, _fn_name, _req_type, _rsp_type)			\
 int __weak								\
-- 
2.31.1

